packages/api/src/index.ts

```diff
@@ -37,3 +38,14 @@ void (async () => {
   .catch((err) => {
     logger.crit('An error happened during app start', err);
   });
+
+
+  app.get('/login/github', (req, res) => {
+    const url = `https://github.com/login/oauth/authorize?client_id=${CLIENT_ID}&redirect_uri=http://localhost:3000/api/auth/callback/github?&scope=user:email`;
+    res.redirect(url);
+  } );
+
+  app.get('/login/github/callback', (req, res) => {
+    const url = ``;
+    res.redirect(url);
```

Comment:
I'm not quite sure as to why this endpoint is here as I don't see it being used anywhere and it's redirecting to an empty url. Is this in place for future use?

Reply:
For now maybe we should just redirect to `/app`?

And if we aren't modifying it, go ahead and put that string directly in the `res.redirect` instead of making it a variable